frontend/src/components/layout/navigation/MobileNavigation.module.scss

```
@@ -0,0 +1,136 @@
+@import "../../../styles/tokens.scss";
+@import "~@mozilla-protocol/core/protocol/css/includes/lib";
+
+.mobile-menu {
+  width: 100%;
+  position: absolute;
+  left: 0;
+  // z-index of 1 (so it overlaps content)
+  // allows menu to slide under main navigation header
+  z-index: 1;
+  transform: translateY(-100%);
+  background: $color-white;
+  flex-direction: column;
+  box-shadow: $box-shadow-lg;
+
+  &.not-active {
+    animation: slide-out 300ms cubic-bezier(0.52, -0.14, 0.31, 1.2);
+    animation-iteration-count: 1;
+    animation-fill-mode: forwards;
+  }
+
+  &.is-active {
+    animation: slide-in 450ms cubic-bezier(0.52, -0.14, 0.31, 1.2);
+    animation-iteration-count: 1;
+    animation-fill-mode: forwards;
+  }
+
+  @media (prefers-reduced-motion) {
+    &.not-active {
+      animation: none;
+    }
+    &.is-active {
+      animation: none;
+    }
+  }
+
+  @keyframes slide-in {
+    100% {
+      transform: translateY(75px);
+    }
+  }
+  @keyframes slide-out {
+    0% {
+      transform: translateY(75px);
+    }
+    100% {
+      transform: translateY(-100%);
+    }
+  }
+}
+
+.menu-item-list {
+  @media screen and #{$mq-md} {
+    display: none;
+  }
+}
+
+.menu-item {
+  display: flex;
+
+  @include text-display-xxs;
+  padding: $spacing-lg $spacing-md;
+  font-family: $font-stack-firefox;
+  border-bottom: 2px solid $color-light-gray-20;
+
+  .link {
+    display: flex;
+    border: none;
+    background-color: transparent;
+    padding: 0;
```

Small suggestion: if you move the padding to the actual links, and make them full width, then the entire menu item is clickable/tappable, rather than just the text within it. (Oh, and maybe you could add a hover style as well? Just in case someone has resized the window on a device with a mouse.)

```suggestion
  font-family: $font-stack-firefox;
  border-bottom: 2px solid $color-light-gray-20;

  .link {
    display: flex;
    border: none;
    background-color: transparent;
    padding: $spacing-lg $spacing-md;
    width: 100%;
```